pkg/http/api.go

```diff
@@ -677,14 +677,14 @@ func (a *api) onCreateActorTimer(reqCtx *fasthttp.RequestCtx) {
 		respondWithError(reqCtx, fasthttp.StatusInternalServerError, msg)
 		log.Debug(msg)
 	} else {
-		respondEmpty(reqCtx, fasthttp.StatusOK)
+		respondEmpty(reqCtx, fasthttp.StatusCreated)
```

Comment:
The semantics of 200 and 201 are slightly different. https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201.

Reply:
Right, I'm just trying to be consistent with what we're doing in other places. If we think the right thing to do is to have all our PUT/POST APIs give 200, I'm fine with that.

Reply:
See above. Giving 204 for all these.